CODE_STYLE.md

```diff
@@ -0,0 +1,57 @@
+## Code style
+> [!IMPORTANT]  
+> All classes do live in the `godot` namespace   
```

Comment:
> All **exported** classes do live in the godot namespace

I can imagine a situation when we will be implementing something in "raw" C++, without Godot classes, then we wrap it in Godot Node class (for example to avoid multiple inheritance).

Reply:
"exported/All classes used in Godot" something like this